doc/reference/devices_disk.md

```diff
@@ -1,58 +1,61 @@
 (devices-disk)=
 # Type: `disk`
 
-Supported instance types: container, VM (supports hotplugging)
+```{note}
+The `disk` device type is supported for both containers and VMs.
+It supports hotplugging for both containers and VMs.
+```
 
-Disk entries are essentially mount points inside the instance. They can
-either be a bind-mount of an existing file or directory on the host, or
-if the source is a block device, a regular mount.
+Disk devices are essentially mount points inside the instance.
```

Comment:
These 2 sentences are only true for containers. Now that we have VMs disks are passed into them very differently.

VMs share host side mount shares via 9p and virtiofs (if available).
Block based disks (volumes) are shared as virtio disks.

Reply:
OK, I tried to rewrite this - please check again. I can't claim that I really understand it. ;)